src/olympia/addons/models.py

```diff
@@ -892,19 +892,19 @@ def initialize_addon_from_upload(cls, data, upload, channel, user):
             )
         if user:
             AddonUser(addon=addon, user=user).save()
-        activity.log_create(amo.LOG.CREATE_ADDON, addon)
+        activity.log_create(amo.LOG.CREATE_ADDON, addon, user=user)
         log.info(f'New addon {addon!r} from {upload!r}')
         timer.log_interval('7.end')
         return addon
 
     @classmethod
     def from_upload(
         cls,
+        *,
         upload,
         selected_apps,
         channel=amo.RELEASE_CHANNEL_LISTED,
-        parsed_data=None,
-        user=None,
```

Comment:
Removed this because this method receives a `FileUpload`, so we should already have a `user`.